lib/widgets/store.dart

```diff
@@ -51,6 +53,28 @@ class GlobalStoreWidget extends StatefulWidget {
     return widget!.store;
   }
 
+  /// The user's [GlobalSettings] data within the app's global data store.
```

Comment:
`GlobalSettings` is the table class; would it be better to refer to `GlobalSettingsStore` since that's what returned by this method?

Reply:
The return type doesn't need a link in the doc, since it's shown directly in the code. By contrast a link to related other types can be helpful for navigating around. (The same reason is why e.g. the doc on the table class `Accounts` mentions the data class `Account`, which doesn't otherwise appear in the code there and so wouldn't otherwise be easy to navigate to.)